README.md

```diff
@@ -174,8 +174,7 @@ maintenance processes.
 *   Javascript: If you don't want to use our version, which depends on Closure,
     there are several other options, including
     https://github.com/halt-hammerzeit/libphonenumber-js (a stripped-down
-    version) and https://github.com/seegno/google-libphonenumber (installable
-    via npm, a browserify-compatible wrapper)
+    version, about 110 KiloBytes in size) and https://github.com/seegno/google-libphonenumber (a browserify-compatible wrapper installable via npm, genuine all-in-one build of the original library, includes Google Closure library, about 420 KiloBytes in size)
```

Comment:
Happy for more detail, but:
- KB not KiloBytes
- wrap at 80 chars
- what does "genuine" or "all-in-one" mean here? Does this add anything? I as a user am more confused now than before.